bugbot/rules/perfalert_inactive_regression.py

```diff
@@ -0,0 +1,155 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import collections
+
+from libmozdata.bugzilla import Bugzilla
+
+from bugbot import logger, utils
+from bugbot.bzcleaner import BzCleaner
+from bugbot.user_activity import UserActivity, UserStatus
+
+
+class PerfAlertInactiveRegression(BzCleaner):
+    def __init__(self, nweeks=1):
+        super().__init__()
+        self.nweeks = nweeks
+        self.extra_ni = {}
```

Comment:
We could pass the number of weeks directly here since it is the same for all bugs:

```suggestion
        self.extra_ni = {
            "nweeks": self.nweeks,
        }
```

We will not need to pass it again in `set_autofix()`. We could also use it directly in the template.